### PR TITLE
Small improvements on 15 minutes dapp

### PR DESCRIPTION
**Pull request by ccorcoveanu**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/tutorials/your-first-dapp.md [hunk 1]
```diff
@@ -71,79 +64,27 @@ First let's create a new folder for our project, I'll name it `ping-pong`.
...
-
-[comment]: # (mx-context-auto)
-
-### **Create an owner wallet**
-
-We now have all the prerequisites installed, let's create an owner's wallet **PEM file**.
+### **Create the owner wallet**
 
-The smart contract can only be deployed on the blockchain by an owner, so we will create an owner wallet [here](https://devnet-wallet.multiversx.com). The owner can also update the contract, later on, if needed.
+The smart contract can only be deployed on the blockchain by an owner, so let's create an owner's wallet **PEM file**. The owner can also update the contract, later on, if needed. Keep in mind we only use PEM wallets for testing and playing around with non-production code. For real applications please follow best practices, use secure wallets that can be generated [here](https://wallet.multiversx.com)
```
Comment (andreibancioiu) [inline docs/developers/tutorials/your-first-dapp.md:76]: Full stop at the end of paragraph.

File: docs/developers/tutorials/your-first-dapp.md [hunk 2]
```diff
@@ -242,6 +184,8 @@ All right, let's move on to the application layer.
 
 ### **Clone the Sample App**
 
+First make sure to go back to the root `ping-pong` folder
```
Comment (andreibancioiu) [inline docs/developers/tutorials/your-first-dapp.md:187]: Full stop at end of paragraph.

File: docs/developers/tutorials/your-first-dapp.md [hunk 3]
```diff
@@ -253,41 +197,35 @@ cd dapp
 
 ### **Configure the app**
 
-Customize the Smart Contract Address
+Use the prefered editor and customize the Smart Contract Address located in `src/config/config-devnet.tsx`
```
Comment (andreibancioiu) [inline docs/developers/tutorials/your-first-dapp.md:200]: Typo "preferred".

Colon is missing.

File: docs/developers/tutorials/your-first-dapp.md [hunk 4]
```diff
@@ -253,41 +197,35 @@ cd dapp
...
-```
-
-Save and close `config.tsx` and we're ready for the first build.
 
 [comment]: # (mx-context-auto)
 
 ### **Build the dApp**
 
+:::important
+[Please make sure you have **yarn installed**](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) on your machine.
```
Comment (andreibancioiu) [inline docs/developers/tutorials/your-first-dapp.md:216]: The link contains the segment for mac. Perhaps remove the segment?

File: docs/developers/tutorials/your-first-dapp.md (hunk 1 above)
Comment (ccorcoveanu) [reply docs/developers/tutorials/your-first-dapp.md:76]: fixed

File: docs/developers/tutorials/your-first-dapp.md (hunk 2 above)
Comment (ccorcoveanu) [reply docs/developers/tutorials/your-first-dapp.md:187]: done

File: docs/developers/tutorials/your-first-dapp.md (hunk 3 above)
Comment (ccorcoveanu) [reply docs/developers/tutorials/your-first-dapp.md:200]: fixed

File: docs/developers/tutorials/your-first-dapp.md (hunk 4 above)
Comment (ccorcoveanu) [reply docs/developers/tutorials/your-first-dapp.md:216]: fixed

**Review by schimih — APPROVED** [review]

**Pull request merged**